runtime/ftplugin/qf_toc.vim

```diff
@@ -0,0 +1,23 @@
+function! s:setup_toc() abort
```

Comment:
Hmm. The other location was better IMO. ftplugin/ should only contain filetypes. `qf_toc` isn't a filetype is it?